### PR TITLE
added solution to lc:602

### PR DESCRIPTION
**Pull request by Dhoni77**
✅ update readme
✅ added solution file

### CODE REVIEW

File: solution/0600-0699/0602.Friend Requests II Who Has the Most Friends/README.md [hunk 1]
```diff
@@ -71,7 +71,19 @@ RequestAccepted 表：
 ### **SQL**
 
 ```sql
-
+SELECT 
+    ids AS id, COUNT(*) cnt
```
Comment (poyea) [inline solution/0600-0699/0602.Friend Requests II Who Has the Most Friends/README.md:75]:
```suggestion
    ids AS id, COUNT(*) num
```

File: solution/0600-0699/0602.Friend Requests II Who Has the Most Friends/README.md [hunk 2]
```diff
@@ -71,7 +71,19 @@ RequestAccepted 表：
...
+FROM
+    (SELECT 
+        requester_id AS ids
+    FROM
+        RequestAccepted UNION ALL SELECT 
+        accepter_id
+    FROM
+        RequestAccepted) t
+GROUP BY ids
+ORDER BY cnt DESC
```
Comment (poyea) [inline solution/0600-0699/0602.Friend Requests II Who Has the Most Friends/README.md:85]:
```suggestion
ORDER BY num DESC
```

**Review by poyea — COMMENTED** [review]
`cnt` -> `num`

File: solution/0600-0699/0602.Friend Requests II Who Has the Most Friends/README.md (hunk 2 above)
Comment (Dhoni77) [reply solution/0600-0699/0602.Friend Requests II Who Has the Most Friends/README.md]: Done the changes @poyea 👍

File: solution/0600-0699/0602.Friend Requests II Who Has the Most Friends/README.md (hunk 1 above)
Comment (Dhoni77) [reply solution/0600-0699/0602.Friend Requests II Who Has the Most Friends/README.md]: Done 👍

**Review by yanglbme — APPROVED** [review]

**Pull request merged**